about/changelog.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Changelog
+excerpt: Changelog
+keywords: [changelog, upgrades, updates, releases]
+---
+
+# Changelog
+
+## TimescaleDB 2.15.2
+
+June 7, 2024
+
+🐛 Bugfixes
+
+- [#6975](https://github.com/timescale/timescaledb/issues/6975): Fix sort pushdown for partially compressed chunks.
+- [#6976](https://github.com/timescale/timescaledb/issues/6976): Fix removal of metadata function and the update script.
+- [#6978](https://github.com/timescale/timescaledb/issues/6978): Fix segfault in compress_chunk with primary space partition.
+- [#6993](https://github.com/timescale/timescaledb/issues/6993): Disallow hash partitioning on the primary column.
+
+## May 2024 Update
+
+May 31, 2024
+
+### 🔍 Database audit logging via the pg_audit extension
```

Comment:
Please avoid latinisms: https://developers.google.com/style/abbreviations#dont-use

Database audit logging with pg_audit

It is a style thing, but I find all these headings break up the information and make it hard to digest. I suggest using a list:

* **Database audit logging with pg_audit** 
   
     The ...

Reply:
Removed `via` from the heading 9dd2537c3b0c0a91dbfec98dc50826624d4aeffc

---

Regarding the headings, I did it for two reasons. 1) so that there's sufficient space for each heading to explain it (ie you can write multiple paragraphs) and 2) you can _link_ to headings